testing/chain.go

```diff
@@ -53,14 +53,14 @@ type SenderAccount struct {
 type TestChain struct {
 	testing.TB
 
-	Coordinator   *Coordinator
-	App           TestingApp
-	ChainID       string
-	LastHeader    *ibctm.Header   // header for last block height committed
-	CurrentHeader cmtproto.Header // header for current block height
-	QueryServer   types.QueryServer
-	TxConfig      client.TxConfig
-	Codec         codec.Codec
+	Coordinator    *Coordinator
+	App            TestingApp
+	ChainID        string
+	LastHeader     *ibctm.Header   // header for last block height committed
+	ProposedHeader cmtproto.Header // header for current block height
```

Comment:
think we should also tweak comment here, one suggestion comes to mind:

```suggestion
	ProposedHeader cmtproto.Header // proposed (uncommitted) header for current block height
```